Dockerfile

```diff
@@ -0,0 +1,32 @@
+# ===============================================
+FROM helsinkitest/node:10-slim as appbase
```

Comment:
Some of the packages I'm using require `node v12`, would it be possible to use it here?

Reply:
Apparently Node v12 is more stable than 11 so i would suggest node v12 also. 
Will require some work to put node 12 img on Hki dockerhub